src/HealthGPS/mapping.cpp

```diff
@@ -7,24 +7,17 @@
 #include <tuple>
 
 namespace hgps {
-hgps::MappingEntry::MappingEntry(std::string name, short level, core::Identifier entity_key,
-                                 FactorRange range, bool dynamic_factor)
-    : name_{name}, name_key_{core::Identifier{name}}, level_{level},
-      entity_key_{std::move(entity_key)}, range_{range}, dynamic_factor_{dynamic_factor} {}
-
-MappingEntry::MappingEntry(std::string name, short level, core::Identifier entity_key,
-                           bool dynamic_factor)
-    : MappingEntry(name, level, entity_key, FactorRange{}, dynamic_factor) {}
 
 MappingEntry::MappingEntry(std::string name, short level, core::Identifier entity_key,
```

Comment:
What on earth are we using `short`s for?! :laughing:

Reply:
Memory efficiency! xD I have no idea.